.github/workflows/build-docs.yml

```diff
@@ -67,11 +65,16 @@ jobs:
           cd build
           cmake -DPOSTGRESQL_VERSION=${{ matrix.psql }} -DCMAKE_BUILD_TYPE=Release -DWITH_DOC=ON -DES=ON -DLINKCHECK="ON" ..
 
-      - name: Check English Links
+      - name: Check Links for English
         run: |
           cd build
           make linkcheck-en
 
+      - name: Check Links for Spanish
```

Comment:
@cvvergara after I understood, that one workflow is for building the documentation (including checking the links) and the other one is just to check the links, how about this?
- the file name says `build-docs`
- the link check is done for all supported languages?